net-wireless/uhd/uhd-3.15.0.0-r100.ebuild

```diff
@@ -61,7 +60,7 @@ src_unpack() {
 }
 
 src_prepare() {
-	cmake-utils_src_prepare
+	cmake_src_prepare
 
 	gnome2_environment_reset #534582
 
```

Comment:
As for sed, is it still needed? "please verify"?

Upstream a patch if it is still needed.

Reply:
https://github.com/EttusResearch/uhd/commit/75b7991378e3f465f9d91cba597d98ee5b9054ed.patch
has been applied upstream to the 4.x branch but no releases have been made to fix this in 3.x.   Dropping 3.15 since despite everything it still doesn't build and upstream said it's not worth keeping.

Reply:
sounds good, thanks